examples/next/pages/ui/hooks/useDataStore/filter/index.page.tsx

```diff
@@ -0,0 +1,56 @@
+import { Amplify, DataStore, Hub } from 'aws-amplify';
+import { Authenticator } from '@aws-amplify/ui-react';
+import * as React from 'react';
+import '@aws-amplify/ui-react/styles.css';
+
+import { AuthSignOutButton } from '../AuthActions';
+import awsExports from '../aws-exports';
+import CollectionAndRecord from './CollectionAndRecord';
+import { initializeTestData } from '../utils';
+
+Amplify.configure({
+  ...awsExports,
+  aws_appsync_graphqlEndpoint: 'https://fake-appsync-endpoint/graphql',
+});
+
+export default function App() {
+  const [isInitialized, setInitialized] = React.useState(false);
+  const initializeStarted = React.useRef(false);
+
+  React.useEffect(() => {
+    (window as any).LOG_LEVEL = 'DEBUG';
```

Comment:
IIRC, you can do 

```ts
import { Logger, /** rest of the imports */ } from 'aws-amplify';

Logger.LOG_LEVEL='DEBUG';
```

Reply:
I'd also consider moving this line above `App`, since this logic isn't related to the component.